src/diff/index.js

```diff
@@ -226,7 +229,8 @@ export function diff(
 				isSvg,
 				excessDomChildren,
 				commitQueue,
-				isHydrating
+                		isHydrating,
+                		doc
```

Comment:
We can remove Lines 39-41 if we inline this (and save bytes)

```suggestion
                		parentDom.ownerDocument
```